hive/indexer/cli.py

```diff
@@ -26,7 +27,8 @@ def indexer():
 def index_from_steemd():
     """import blocks from .json.lst file"""
     echo('Loading blocks from steemd...')
-    sync_from_steemd()
+    setup()
+    sync_from_steemd(True)
```

Comment:
`index_from_steemd` is outdated, if it's meant to be the main "launch daemon" command, let's rename it and just have it call `run()` method in hive.indexer.core, runs `setup()` only if needed, handles multiple sync strategies, and then streams blocks forever.